hudi-common/src/main/avro/HoodieClusteringPlan.avsc

```diff
@@ -30,7 +30,7 @@
     },
     {
        "name":"strategy",
-       "type":["HoodieClusteringStrategy", "null"],
+       "type":["null", "HoodieClusteringStrategy"],
```

Comment:
oops. cc @satishkotha @n3nash  you may also want to look at this. default values work only based on the type of the first element in an union IIUC.

Reply:
ack. thanks for fixes.